autoload/OmniSharp/locations.vim

```diff
@@ -55,17 +55,61 @@ function! OmniSharp#locations#Preview(location) abort
   endif
 endfunction
 
-function! OmniSharp#locations#SetQuickfix(list, title)
+function! OmniSharp#locations#SetQuickfix(list, title) abort
+  call s:SetQuickfixFromDict(a:list, {'title': a:title})
+endfunction
+
+function! OmniSharp#locations#SetQuickfixWithVerticalAlign(list, title) abort
+  call s:SetQuickfixFromDict(a:list, {'title': a:title, 'quickfixtextfunc': 's:QuickfixVerticalAlignFunction'})
```

Comment:
There is an issue with using an `s:Function` scoped function with `quickfixtextfunc`: The function is not available outside the script so cannot be used next time the quickfix window is opened.

To demonstrate, try closing the quickfix window, then re-opening it with `:copen`. I get an error.

I think we'll have to use something like `OmniSharp#locations#_quickfixtextfuncAlign()` instead of `s:QuickfixVerticalAlignFunction()`

Reply:
Done